retail/interactive-tutorial/RetailPredictions.Samples/PredictionWithParametersSample.cs

```diff
@@ -0,0 +1,144 @@
+﻿// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START retail_prediction_with_parameters]
+// Get predictions from catalog using Retail API
+
+using Google.Cloud.Retail.V2;
+using Google.Protobuf.WellKnownTypes;
+using Runner.Attributes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace RetailPredictions.Samples
+{
+    /// <summary>
+    /// The prediction with parameters sample class.
+    /// </summary>
+    public class PredictionWithParametersSample
+    {
+        private const string Endpoint = "retail.googleapis.com";
+
+        private static readonly string ProjectNumber = Environment.GetEnvironmentVariable("PROJECT_NUMBER");
```

Comment:
We are not doing this as part of the sample, please take a look at the rest of Restail sample that have been merged and follow those as a template.

Reply:
I Updated this code